### ISSUE TITLE
complete DSA course buddy

### ISSUE THREAD

**Issue by HimanshuJakhmola356** [kunal-kushwaha/DSA-Bootcamp-Java#694]
i am following that path , and worrying about not geting the same type of teaching and explination anywhere so please complete the course you hangon stack and queues littreally its a good gap but now you must Continue ,and i also email you regarding this but didn't get reply..

**Comment by lubnafathima** [comment]
Wait so this course isn't completed?
My placements are starting soon
will this course be suitable then?

**Comment by ShashankRoy27** [comment]
No, not suitable if it isn't complete. What you can do however, is do the
topics that are taught by Kunal from his DSA playlist and the other topics
from other platforms like GFG, any YouTube channel or even the NADOS
website of Pepcoding.

**Comment by suraj-996** [comment]
Just do whatever he has taught already and solve questions related to that topic on leetcode as much as you can by doing that you will build more confidence on that and when he will start again teaching then you can follow him or you can learn from other platform also.

**Comment by lubnafathima** [comment]
That's the plan! 
Thank you 😀

**Comment by sonustar** [comment]
Just follow what he taught and practice as much as you can...

**Comment by lubnafathima** [comment]
Will Do!
Thank you

**Comment by kapilepatel** [comment]
@HimanshuJakhmola356  I believe your question was answered, can you close the issue?

**Comment by Aditya-Jaiswal0** [comment]
> He will complete it this year, he tweeted. Don't worry.

It would be nice if you can show his tweet.

**Comment by SensationalShubham** [comment]
He still didn't even upload 1 single video.
